src/products/firefox_desktop/fluent_migrations.md

```diff
@@ -0,0 +1,83 @@
+# Running Fluent migrations
+
+[Fluent migrations](https://firefox-source-docs.mozilla.org/l10n/migrations/index.html) are used to migrate existing translations from legacy formats (DTD, properties) to Fluent, or existing Fluent messages to new messages or different FTL files. This document describes in detail which operations need to be performed in order to run migrations on over 150 Mercurial repositories for Firefox. Several access rights are required:
+* Write access to [l10n-central](https://hg.mozilla.org/l10n-central), i.e. SSH access with `scm_l10n` rights.
+* Sign-off rights in [Elmo](https://l10n.mozilla.org/).
+* Admin access to [Pontoon](https://pontoon.mozilla.org/).
+* An account on Heroku with access to [Pontoon apps](https://dashboard.heroku.com/apps/mozilla-pontoon/resources).
+
+## Prerequisites
+
+In order to run migrations:
+* Install the Python packages [fluent.migrate](https://pypi.org/project/fluent.migrate/) and [compare-locales](https://pypi.org/project/compare-locales/).
```

Comment:
Just `fluent.migrate` is enough, it comes with a package dependency on `compare-locales`. Also makes sure you're installing the right one.